JS/lesson-16/index.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+// let admin;
+// let userName = 'Jon';
+// admin = userName;
+// alert(admin);
+
+// let ourPlanet;
+// let userName;
+
+// let x = 20;
+// let y = 58;
+// let z = 42;
+// let rezult = x+y+z;
+// console.log(rezult);
+
+// let sec = 60;
+// let minute = 60;
+// let hour = 24;
+// let day = 365;
+// let myAge = prompt('Vvedite Vash vozrast');
+// myAge = Number(myAge);
+// let myAgeInSeconds = myAge*day*hour*minute*sec;
+// alert(`Vash vozrast v secundah - ${myAgeInSeconds}`);
+
+// let count = 42;
+// let userName = '42';
+// count = String(count);
+// userName = Number(userName);
+// console.log(count,userName);
+
+// let a = 1;
+// let b = 2;
+// let c = "белых медведей";
+// alert(`${a}${b} ${c}`);
+
+// let a = 'доступ';
+// let b = 'морпех';
+// let c = 'наледь';
+// let d = 'попрек';
+// let f = 'рубило';
+// let lengthWords = a.length+b.length+c.length+d.length+f.length;
```

Comment:
Как вариант можно было сократить запись
```suggestion
// const lengthWords = (a + b + c + d + f).length;
```